pkg/identity/identity.go

```diff
@@ -109,6 +109,11 @@ func (id *Identity) StringID() string {
 	return id.ID.StringID()
 }
 
+// StringID returns the identity identifier as string
+func (id *Identity) String() string {
```

Comment:
Is this meaningfully distinct from `StringID()`? Also, the comment refers to `StringID()`, not `String()`.

Reply:
It makes `*Identity` implement the `Stringer` interface, which allows for more readable debug logs when using `%s` for the identity. I can add other fields as well, including labels.

Reply:
This response satisfies the `Stringer` interface. ;)

(In all seriousness, for CIDR identities the labels may become quite cumbersome, not sure whether it would actually make it easier if more fields are included)

Reply:
I'm ambivalent. If we find that we need more information from logging, we can always add it later if need be?